pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -137,6 +111,8 @@ type PipelineTriggerRef struct {
 // PipelineRunStatus defines the observed state of PipelineRun
 type PipelineRunStatus struct {
 	Conditions duckv1alpha1.Conditions `json:"conditions"`
+	// In #107 should be updated to hold the location logs have been uploaded to
+	Results Results `json:"results"`
```

Comment:
The results field in the pipelineRun is optional but here in the status it not, what happens if there is a target url for the results?

Reply:
we can make this optional too!